lms/djangoapps/grades/tests/test_new.py

```diff
@@ -365,19 +458,7 @@ def test_weight_metadata_alterations(self, alterations, expected_earned, expecte
     )
     @ddt.unpack
     def test_graded_metadata_alterations(self, alterations, expected_earned, expected_possible):
-        score = self._get_score_with_alterations(alterations)
+        self._add_problem_with_alterations(alterations)
+        score = self._get_score()
         self.assertEqual(score.graded_total.earned, expected_earned)
         self.assertEqual(score.graded_total.possible, expected_possible)
-
-    @ddt.data(
-        {u'max_score': 99.3},
-        {u'max_score': 1.0},
-        {u'max_score': 0.0},
-        {u'max_score': None},
-    )
-    def test_max_score_does_not_change_results(self, alterations):
```

Comment:
It turned out that max score is calculated from the number of parts the problem has, not from an xblock field, so I was creating a false field that just got ignored.  Might as well test that the "wild_rumpus" field doesn't change the result.